docroot/profiles/custom/cgov_site/modules/custom/cgov_image/src/CgovImageTools.php

```diff
@@ -25,6 +25,7 @@ public function findCropByStyle($image_style) {
 
       case "cgov_featured":
       case "cgov_social_media":
+      case "ncids_collections_media_4x3":
```

Comment:
How was this not needed before with the dynamic lists? This is an actual question, not a fix this. I see the dynamic list on news & events is using ncids_collections_media_4x3.

Reply:
We are using this to dictate which placeholder image to use in our twig files. When developing dynamic lists, I think we didn’t need to account for images because the content editor would check that internally linked items would have its image.